protocol/tbdex_schema_examples.js

```diff
@@ -0,0 +1,50 @@
+let offering = {
+  "pair": "BTC_USD",
+  "unitPrice": 27000.00,
+  "fee": 1.00,
+  "min": 10.00,
+  "max": 100.00,
+  "presentationRequest": { },
+  "payinInstruments": [{
```

Comment:
is it possible that payment instruments (pay in or pay out) can affect the unit price? if that's the case, then should we have one payin/payout instrument combo per offer, instead of having arrays of all accepted payin/payout instruments per offer?

Reply:
No I don't think that's ever the case. I do think different payment types offer different fee amounts, but a unit price is a universal number.